src/ContractInteractor.ts

```diff
@@ -919,20 +945,25 @@ export default class ContractInteractor {
 
     async getERC20Token(
         address: string,
-        options?: TokenOptions
-    ): Promise<Token> {
-        const token = await this._createERC20(address);
+        options?: Partial<ERC20Options>
```

Comment:
I removed the Partial since you are right, it was already declared before. 

Regarding the optional for options, its because that parameter can be omitted to get just the instance. 

With the last part, was to give some flexibility about the properties that want to be retrieve at the moment of the execution, instead of calling the method property when is needed.